doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -278,6 +279,12 @@ subscription creation/update time. Check details in the document about [initial
 
 [Top](#top)
 
+## Oneshot subscriptions
+
+Apart from the `status` values defined for subscription in the NGSIv2, Orion also allows to use `oneshot`. Please find details in this [Oneshot_Subscriptions](oneshot_subscription.md)
```

Comment:
Sorry to insist on this...

"in the NGSIv2" is a bit weird. Please change it to "in the NGSIv2 specification" (my favourite) or "in NGSIv2" (removing the "the").

In addition note how the final sentence renders:

![imagen](https://user-images.githubusercontent.com/1534240/49391214-55352200-f72b-11e8-99d4-1ab8857c3dc8.png)

"in this [Oneshot_Subscriptions]" is a bit ugly... It would be better to see "in [this piece of documentation]" (my initial suggestion) or "in [the oneshot subscription document]".

Reply:
fixed in 06f6526